src/Orleans/project.json

```diff
@@ -0,0 +1,67 @@
+﻿{
```

Comment:
project.json is real project file and it is portable . The xproj is not needed and you can develop a coreclr project without Visual Studio.
Check this out:
http://dotnet.readthedocs.org/en/latest/dnx/overview.html
Also it defines the nugget package.